.github/workflows/upload-to-pypi.yml

```diff
@@ -0,0 +1,106 @@
+name: Upload-ansible-release-artifacts-to-pypi
+on:
+  workflow_dispatch:
+    inputs:
+      ansible-version:
+        description: >-
+          Release Version. Example : 11.1.0
+        required: true
+      ansible-major-version:
+        description: 'Example 11'
+        required: true
+
+env:
+  BUILD_DATA_DIR: "antsibull/build/ansible-build-data/${{ inputs.ansible-major-version }}"
+  BUILDFILE: "ansible-${{ inputs.ansible-major-version }}.build"
+  DEPSFILE: "ansible-${{ inputs.ansible-version }}.deps"
```

Comment:
Are these env vars used? It doesn't look like it.

Reply:
Yes, I was not using them Thank you for pointing it out. Removed it now.